### DISCUSSION TITLE
Is it possible to disable the cdn / netlify resources in production?

### DISCUSSION THREAD

**Discussion by OCram85** [thuliteio/doks#215] [Q&A]
Is it possible to disable the cdn delivered resources like fonts, js, images in production builds?
I would like to use this project in an air gapped environment behind a corporate proxy. 😠

**Comment by h-enk** [comment]
Which CDN delivered resources?

**Reply by h-enk** [reply]
All Doks resources are served from your production domain (and that's on purpose).

**Reply by OCram85** [reply]
Srry for the missing details:

- I changed some content after cloning this repo in a new project
- used npm run buil / hugo --gc for production build
- Tried to serve the publish folder via caddy

And then noticed some missing parts in the html header like the fonts which points to a netlify url...

**Reply by h-enk** [reply]
Ah, that helps. Did you update `baseurl` in `./config/production/config.toml`?

**Reply by OCram85** [reply]
I changed some highlight.js language settings and page content, but didn't touch the baseurl:
```
baseurl = "/"
```

**Reply by h-enk** [reply]
The default `baseurl` in `./config/production/config.toml` is: `"https://doks.netlify.app/"`

**Reply by OCram85** [reply]
![](https://media0.giphy.com/media/xT5LMzIK1AdZJ4cYW4/giphy.gif)

Srry, I absolutely missed this 😿

**Reply by h-enk** [reply]
No worries!
